src/web_app/html/index_template.html

```diff
@@ -22,6 +22,7 @@
 
   <title>AppRTC</title>
 
+  <link rel="manifest" href="manifest.json">
```

Comment:
@samdutton could this be cause of #181 ? If it's for some reason requested when the browser is on https://apprtc.appspot.com/r/roomname, would that result it the base URL being `https://apprtc.appspot.com/r/`rather than `https://apprtc.appspot.com/` ?